### PR TITLE
Add support for ignored tests.

### LINKED ISSUE #2: Ignored test marked as failed.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #2.

### CODE REVIEW

File: composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt [hunk 1]
```diff
@@ -26,7 +32,18 @@ data class InstrumentationEntry(
         val stack: String,
         val statusCode: Int,
         val timestampNanos: Long
-)
+) {
+    companion object {
+
+        // See `android.support.test.internal.runner.listener.InstrumentationResultPrinter`.
```
Comment (ming13) [inline composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt]: Javadoc?

File: composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt [hunk 2]
```diff
@@ -26,7 +32,18 @@ data class InstrumentationEntry(
...
+
+        const val REPORT_VALUE_RESULT_START = 1
+        const val REPORT_VALUE_RESULT_OK = 0
+        const val REPORT_VALUE_RESULT_FAILURE = -2
+        const val REPORT_VALUE_RESULT_IGNORED = -3
+        const val REPORT_VALUE_RESULT_ASSUMPTION_FAILURE = -4
```
Comment (ming13) [inline composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt]: `enum`?

Comment (artem-zinnatullin) [inline composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt]: I wanted, but then though that there are cases when it's not appropriate, but actually it is, will do

File: composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt (hunk 1 above)
Comment (artem-zinnatullin) [inline composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt]: For each const?

Comment (ming13) [inline composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt]: Just for the class link.

File: composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt (hunk 2 above)
Comment (artem-zinnatullin) [inline composer/src/main/kotlin/com/gojuno/composer/Instrumentation.kt]: Done, much better now, thanks

**Pull request merged**